sdk/confidentialledger/azure-confidentialledger/CHANGELOG.md

```diff
@@ -1,12 +1,23 @@
 # Release History
 
-## 1.0.0b2 (Unreleased)
+## 1.0.0 (Unreleased)
```

Comment:
Do we plan to populate this just before the release goes out?

Reply:
Per Isabella, we'll populate this when we have a release date settled.